### DISCUSSION TITLE
Will anyone use YOLO11 on the web?

### DISCUSSION THREAD

**Discussion by bbhxwl** [microsoft/onnxruntime#22447] [API Q&A]
Will anyone use YOLO11 on the web?

**Comment by xy3xy3** [comment]
I also want to know

**Comment by xy3xy3** [comment]
https://github.com/Hyuto/yolov8-onnxruntime-web/tree/master
I have tried to used yolov11n.onxx in this project.And it successfully works.
![image](https://github.com/user-attachments/assets/e3f96004-a9f5-4140-8862-3387dce3dc16)
![image](https://github.com/user-attachments/assets/0c549738-7f50-4923-8cca-41c3807949e1)
